standards/layout.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Layout
+description: Standardizes logic component form-factor and IO gate markings
+---
+
+# Layout
+This is the main logic component layout standard for the SMLC.
+
+## New global terminology
+These are new terms that are used in this standard that should be made global, which will happen in a seperate proposal in the terminology repo.
+- A **component** is a Scrap Mechanic creation intended to be re-used inside of other logic creations.
+
+## Terminology
+- An **interface** is a set of logic gates that form an input or output of a creation (and components, which are creations)
+- An **internal** gate is the opposite of an interface gate, so these are gates that are NOT supposed to be connected to external circuitry.
+- A **flatpak** formfactor means that the component is one block tall, and thus entirely flat.
```

Comment:
typo: `flatpack` should be spellt with a "c"

Reply:
thx